cardano-cli/src/Cardano/CLI/Types/Errors/ShelleyTxCmdError.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE ViewPatterns #-}
```

Comment:
Please do not use `ViewPatterns`. It makes the code less readable.

Reply:
removed